pkg/certificates/reconciler/certificates.go

```diff
@@ -41,8 +41,17 @@ const (
 	expirationInterval   = time.Hour * 24 * 30       // 30 days
 	rotationThreshold    = 10 * time.Minute
 
+	// certificates used by control elements such as autoscaler, ingress controller
 	controlPlaneSecretType = "control-plane"
-	dataPlaneSecretType    = "data-plane"
+
+	// certificates used by trusted data pipeline elements such as activator, ingress gw
+	dataPlanePipelineSecretType = "data-plane-pipeline"
+
+	// certificates used by edges acting as senders and receivers in the data-plane such as queue
+	dataPlaneEdgeSecretType = "data-plane-edge"
```

Comment:
Not really, since it one day can be a sender as well,  and we have no rule right now that the only sender is queue..

Reply:
Im also not super happy with `edge`. It does not really say anything about who is using it. Maybe we can say `Kservice` (thats what would use it, no matter which container in it, right)?

Reply:
It should not say who is using it, it just indicates that the entity using it is an edge that can be a sender or a receiver of data. 
It may or may not be a Kservice. 
It certainly can be a microservice deployed outside of Knative that sends data!
It can maybe also be any event source, but we will look into that when we analyze eventing...

Reply:
I would rather have "data-plane-user" for anything that links to user created components or external components. 
Another option is "data-plane-service", even an event source can be seen as a service, same as Istio has for external services (serviceEntry). Edge is linked to other stuff nowadays, it is an option but would not be my first.

Reply:
Obviously, I think "edge" is best - it takes a very typical networking approach of edges connected with a routing system. 

Anyhow "user" can also do. "service" is very confusing, I can consider the routing/ingress=activator as the service instead. So I would not suggest using "service".

Reply:
See above